docs/_articles/en/apex/writing.md

```diff
@@ -21,13 +21,17 @@ If you want to change how suggestions are pre-selected, see [IntelliSense - Sugg
 
 Code snippets are available for scaffolding class and interface definitions, and a variety of statements such as loops and conditional statements, and so on. When you’re working in an Apex class or trigger, run `Insert Snippet` from the Command Palette to view the available snippets. You can also view these code snippets as code completion suggestions.
 
+Out of the box snippets for Salesforce development are available in these repositories:
+- [Apex Code Snippets](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/snippets/apex.json)
+- [Code Snippets for LWC development](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-lwc/snippets/lwc.json) 
+
 If you want to define your own code snippets, read [Snippets in Visual Studio Code](https://code.visualstudio.com/docs/editor/userdefinedsnippets).
 
-## Code Smartness for SObjects
+## Intellisense for SObjects
 
-To ensure that the code smartness feature correctly prompts completion suggestions, you must refresh the SObject definitions. Run the `SFDX: Refresh SObject Definitions` command from the Command Palette.
+To ensure that the intellisense feature correctly prompts completion suggestions, you must refresh the SObject definitions. Run the `SFDX: Refresh SObject Definitions` command from the Command Palette.
```

Comment:
What's the capitalization for "intellisense"? Feature names tend to be capitalized.

Reply:
you are right about caplitalization. It's a term specific to VS Code and code completion support it offers. Making the change. Thanks!